moment_kinetics/test/fokker_planck_time_evolution_tests.jl

```diff
@@ -287,20 +288,24 @@ function run_test(test_input, expected, rtol, atol, upar_rtol=nothing; args...)
     end
 
     # Convert keyword arguments to a unique name
-    name = input["run_name"]
+    function stringify_arg(key, value)
```

Comment:
Should this function be in this test? Why not in utils.jl?

Reply:
Ah, it could be now that it's used in a few places. Some of the tests have slightly different hacked versions to get short enough names to not cause crashes when passed to HDF5. You're welcome to consolidate them if it bothers you.

Reply:
This seems reason enough to leave it alone : )